go.sum

```diff
@@ -1146,6 +1146,8 @@ google.golang.org/grpc v1.40.0/go.mod h1:ogyxbiOoUXAkP+4+xa6PZSE9DZgIHtSpzjDTB9K
 google.golang.org/grpc v1.45.0/go.mod h1:lN7owxKUQEqMfSyQikvvk5tf/6zMPsrK+ONuO11+0rQ=
 google.golang.org/grpc v1.55.0 h1:3Oj82/tFSCeUrRTg/5E/7d/W5A1tj6Ky1ABAuZuv5ag=
 google.golang.org/grpc v1.55.0/go.mod h1:iYEXKGkEBhg1PjZQvoYEVPTDkHo1/bjTnfwTeGONTY8=
```

Comment:
Should we leave this here as `go mod tidy` isn't working?

Reply:
Yeah, I think it's fine. The checksum entries are only used to validate that code for that package wasn't modifies. It will be cleaned up on the next successful run (once we upgrade other packages as well).